appserver/admingui/payara-console-extras/src/main/java/fish/payara/admingui/extras/rest/PayaraRestApiHandlers.java

```diff
@@ -574,10 +574,12 @@ public static void updateNotifiers(HandlerContext handlerCtx) {
         boolean forRequestTracing = false;
         boolean forHealthCheck = false;
         boolean forMonitoring = false;
+        boolean forAdminAudit = false;
```

Comment:
No pressing need to explicitly assign `false`, as this is the default value already.

Reply:
I know that it is false by default , but I wanted it to match the others and be explicit.